TrajectoryPlannerROS:
  max_vel_x: 1.5
  min_vel_x: -0.4
  max_vel_theta: .7
  min_vel_theta: -0.7
  min_in_place_vel_theta: -0.4

  acc_lim_theta: 1
  acc_lim_x: 2.0
  acc_lim_y: 0

  # Goal Tolerance Parameters
  xy_goal_tolerance: 0.10
  yaw_goal_tolerance: 0.05

  holonomic_robot: false
  meter_scoring: true

  # Forward Simulation Parameters
  sim_time: 1.6
  vx_samples: 18
  vtheta_samples: 20
  sim_granularity: 0.05
  heading_lookahead: 1.0
  path_distance_bias: 5.0
  goal_distance_bias: 1.0
  occdist_scale: 0.02
